oemof/solph/components.py

```diff
@@ -377,6 +384,7 @@ def _storage_balance_rule(block, n, t):
             expr += block.capacity[n, t]
             expr += - block.capacity[n, t-1] * (
                 1 - n.loss_rate[t])
+            expr += n.loss_constant[t]
```

Comment:
For the loss_rate, relative losses are inherently needed. I think, it is easier here, to have absolute values.

For invest storages, it would even make sense to have to kinds of "fixed" losses: One type growing with the nominal capacity, one that is really fixed.

Reply:
Hm... especially in case of invest storages, I think a capacity dependent fix-loss makes sense. In case of a water tank, whose volume is growing and with the volume the surface, then the fix-losses are growing as well. So, in case if no investment in the storage is done, there will be no fix-losses as well.